QUICKSTART.md

```diff
@@ -31,10 +32,34 @@ configuration must be provided by **Applications** which should also depend on t
 libraries will obtain a real implementation only if the user application is configured for it. For
 more details, check out the [Library Guidelines].
 
+## Setting up the SDK
+
+To get started, setup the OpenTelemetry SDK - note that if you are using a different implementation 
+of the OpenTelemetry API, you should refer to the configuration instructions for that SDK. Also, if
+you are using the Auto-Instrumentation Agent, the SDK is configured by the agent itself and you
+should not setup your own.
+
+You can setup a basic configuration that exports traces to a logging stream like this:
+
+```java
+OpenTelemetry openTelemetry = OpenTelemetrySdk.newBuilder()
```

Comment:
Is there any reason to not use the auto-detected path as the default? i.e. I'd prefer we mention first the 54-56 lines - and then as the more detailed version (which is something that in general more advanced users will use, such as the Auto-Instrumentation).